programs/programs/ma/cpp/calculator.py

```
@@ -0,0 +1,69 @@
+    Source: https://earlychildhoodcambridge.org/cpp/
+    """
+
+    amount = 1  # Value varies (free tuition); frontend displays "Varies"
+    eligible_city = "Cambridge"
+    hud_county = "Middlesex"  # Cambridge is in Middlesex County, MA
+    # 65% AMI is not available from the HUD API (MTSP supports 20/30/40/50/60/70/80/100%).
+    # Using 60% MTSP as a conservative approximation.
+    max_ami_percent = "60%"
+    min_child_age = 3
+    max_child_age = 4
+    dependencies = ["income_amount", "income_frequency", "household_size", "county"]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)